e2e_tests/tests/per_provider/normal_tests/create_destroy_key.rs

```diff
@@ -2,28 +2,37 @@
 // SPDX-License-Identifier: Apache-2.0
 use e2e_tests::TestClient;
 use parsec_client::core::interface::requests::ResponseStatus;
+use parsec_client::core::interface::requests::Result;
+#[cfg(not(feature = "cryptoauthlib-provider"))]
 use picky_asn1_x509::RSAPublicKey;
 
 #[test]
-fn create_and_destroy() {
+fn create_and_destroy() -> Result<()> {
```

Comment:
Not returning anything and using `unwrap` is actually easier as with `unwrap` you can see exactly which line panicked but, when returning errors, you don't know which line returned it 😬

Reply:
Well, `Result<()>` will be removed.